tests/LinAlg/matrixTestsSymSparse.hpp

```diff
@@ -118,64 +118,64 @@ class MatrixTestsSymSparse : public TestBase
   }
 
 
-  /**
-   * Block of W += alpha*A
-   *
-   * Precondition: W is square
-   * 
-   * @todo Change parameter _A_ to be of abstract class hiopMatrixSymSparse
-   * as soon as this interface exists.
-   */
-  bool matrixAddToSymDenseMatrixUpperTriangle(
-    hiop::hiopMatrixDense& W,
-    hiop::hiopMatrixSparse& A, // sym sparse matrix
-    const int rank=0)
-  {
-    const local_ordinal_type N_loc = W.get_local_size_n();
-    const local_ordinal_type A_M = A.m();
-    const local_ordinal_type A_N_loc = A.n();
-    assert(W.m() == W.n());
-    assert(W.m() >= A.m());
-    assert(W.n() >= A.n());
```

Comment:
This test should be kept and modified to test `addUpperTriangleToSymDenseMatrixUpperTriangle`.